packages/nextjs/src/server/withClerkMiddleware.ts

```diff
@@ -44,6 +45,10 @@ export const withClerkMiddleware: WithClerkMiddleware = (...args: unknown[]) =>
 
   const proxyUrl = opts?.proxyUrl || PROXY_URL;
 
+  if (!!proxyUrl && !isHttpOrHttps(proxyUrl)) {
```

Comment:
🙃 i think we `isHttpOrHttps(proxyUrl)` covers the case of falsy proxyUrl. Do we need `!!proxyUrl &&` check ?

Reply:
@dimkl  isHttpOrHttps will return false if proxyUrl is undefined, but in this case I want  to throw the error if the proxyUrl was set but it was set with the wrong format.

Reminder:  proxyUrl is something optional